### PR TITLE
Update Branding Strings

### PR DESCRIPTION
**Pull request by linuxwolf**
Updates all the branding strings from Lockbox to Lockwise.  Also updates titles to be just the product, and (to match mobile designs) all instances of "entry" or "entries" to "login" or "logins" (respectively).

**NOTE**  the following are **unchanged**:
* telemetry identifiers (`lockboxv1` telemetry category, `lockbox-addon` prefs entrypoint)
* FAQ link targets (requires updates to content, which can be done in a way that preserves these)
* Survey/feedback links
* Icons (follow up PR once assets are available)

Connected to #206

## Testing and Review Notes

**Requires Manual Inspection/Verification**

## To Do

- [ ] double check the original issue to confirm it is fully satisfied
- [x] add testing notes and screenshots in PR description to help guide reviewers
- [x] request the "UX" team perform a design review (if/when applicable)
- [x] make sure CI builds are passing (e.g.: fix lint and other errors)

### CODE REVIEW

File: src/locales/en-US/list.ftl [hunk 1]
```diff
@@ -121,13 +124,13 @@ intro-page-footer-copy =
   you can sync your information to this device by signing in to or
   creating a { -fxaccount-brand-name }. <go>Learn More</go>
 
-item-details-heading-new = Create New Entry
-item-details-heading-edit = Edit Entry Details
+item-details-heading-new = Create New login
+item-details-heading-edit = Edit login Details
```
Comment (6a68) [inline src/locales/en-US/list.ftl]: Should this be capitalized, "Login"?

Comment (linuxwolf) [reply src/locales/en-US/list.ftl]: d'oh, yeah it should

**Comment by linuxwolf** [comment]
@6a68 
> However, searching for lockbox in the repo does turn up quite a few hits; the website links are probably the most important, but we can defer those if there's a separate ticket for updating the website. @linuxwolf what do you want to do with the leftovers?

Most of the website hits are a good callout, and we can update them now since https://lockwise.firefox.com is up and running (so to speak).  Will need to reconcile the targets that say "Lockbox", though.  That I'll make sure to create an issue for, and keep some amount of "backwards compatibility" (e.g., add an explicit `<a name="....">`).

The icons we're still waiting for, though.  I don't think it's worth creating follow up issues; we can check off the boxes in #206.

**Comment by 6a68** [comment]
Sounds good! :+1: from me, self-merge at your leisure

**Comment by linuxwolf** [comment]
CC @irrationalagent 

A couple of open questions/confirmations:

1. The telemetry identifier is currently `lockboxv1`.  That should _**not**_ change, correct?
2. When opening Sync preferences, we use the telemetry identifier `lockbox-addon`.  Should that change to `lockwise-addon` or remain as-is?

**Comment by irrationalagent** [comment]
> CC @irrationalagent
> 
> A couple of open questions/confirmations:
> 
>     1. The telemetry identifier is currently `lockboxv1`.  That should _**not**_ change, correct?

Correct, its ok to keep it the same. We will be filtering for webextensions only on the analysis side, so the data won't be confused with the bootstrapped addon.

>     2. When opening Sync preferences, we use the telemetry identifier `lockbox-addon`.  Should that change to `lockwise-addon` or remain as-is?

Are you talking about the value for `entrypoint`? if yes, let's stick with lockbox since that's what our historical data for the other apps is (and will remain)

**Comment by linuxwolf** [comment]
> Are you talking about the value for `entrypoint`? if yes, let's stick with lockbox since that's what our historical data for the other apps is (and will remain)

Yes, it's the value for `entrypoint`.  Will keep it as-is then.

**Comment by linuxwolf** [comment]
> Worth a version bump now or soon too?

Since we've followed the "increment version just before release" model here, I think it's too soon.

**Pull request merged**